yt_dlp/extractor/manoto.py

```diff
@@ -0,0 +1,125 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor, traverse_obj
+
+
+_API_URL = 'https://dak1vd5vmi7x6.cloudfront.net/api/v1/publicrole/{}/{}?id={}'
+EXT = 'mp4'
```

Comment:
`EXT` must be inlined. Making just `'mp4'` as a constant is completely meaningless.

Reply:
Done.